commands/alpha/rpkg/pull/command.go

```diff
@@ -113,7 +113,7 @@ func (r *runner) runE(_ *cobra.Command, args []string) error {
 		return errors.E(op, err)
 	}
 
-	if err := util.AddResourceVersionAnnotation(&resources); err != nil {
+	if err := util.AddRevisionMetaData(&resources); err != nil {
```

Comment:
Nit: We should use consistent naming, so this should probably be `AddRevisionMetadata`.